styles.css

```diff
@@ -527,6 +518,130 @@ ul {
     color: #5e6c84;
     border: 1px solid #c1c7d0;
   }
+
+  .popup {
+    display: flex;
+    border-radius: 16px;
+    flex-direction: column;
+    align-items: flex-start;
+    justify-content: space-between;
+    width: 90%;
+    z-index: 100;
+    background-color: white;
+    position: fixed;
+    top: 40px;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    margin: auto;
+    margin-bottom: 40px;
+  }
```

Comment:
Good job updating the design of the popup window on the mobile version, as suggested by the previous reviewer👏.  However, currently, the full contents of the popup window cannot be seen on the mobile version on devices with smaller heights(iPhone SE, for example) as they overflow the parent container🪲. Kindly fix this by allowing your popu window to scroll so all the content can be seen🙏

<img width="172" alt="image" src="https://user-images.githubusercontent.com/37116322/225739913-d45f7694-4012-405f-aa4f-18d1f20f7f72.png">

Reply:
Thanks for your comment. I just wanted to ask. Should I just add "overflow: auto" to this class and this will make the page in iPhone SE screen vertically scrollable ? Thanks